frontend/src/views/Layout/ContentFooter.vue

```diff
@@ -59,6 +66,11 @@ export default {
     return {
       year: new Date().getFullYear(),
       version: CONFIG.APP_VERSION,
+      hash: process.env.VUE_APP_BUILD_COMMIT,
+      shortHash:
+        process.env.VUE_APP_BUILD_COMMIT !== undefined
+          ? process.env.VUE_APP_BUILD_COMMIT.substr(0, 8)
```

Comment:
I do not like that you read from the `process.env` directly. I tend to capsulate the access to the environment variables in the config to make it easy to understand what config options are available and have one central point to influence them. Furthermore you have the typescript hints and other evaluation methods to ensure you only have valid values. It furthermore allows you to control defaults and you do not do that in the code, but centralized in the config js file.

Reply:
I will try it.